src/components/preprocessing/curator.py

```diff
@@ -5,16 +5,15 @@
 import logging
 
 from .text_curator import TextCurator
-from .table_curator import TableCurator
 
 logger = logging.getLogger(__name__)
-NAME_CLASS_MAPPING = {"TextCurator": TextCurator, "TableCurator": TableCurator}
+NAME_CLASS_MAPPING = {"TextCurator": TextCurator}
```

Comment:
Could we keep the table curator import and mapping here or is it absolutely necessary to delete it? Afaik the pdf_table_curator notebook needs the TableCurator class in order to work.